test/integration.test.ts

```diff
@@ -13,18 +14,16 @@ import { AnyValidateFunction } from "ajv/dist/core.js";
 const Ajv = _Ajv.default;
 const addFormats = _addFormats.default;
 
-if (!process.env.AUTH_TOKEN) {
-  console.error("No AUTH_TOKEN env var set. Exiting.");
-  process.exit(1);
-}
+const ZT_VERSION = process.env.ZT_VERSION || "1.12.2"
 
-const isNextApiVersion = process.env.ENABLE_UNSTABLE;
+function compareVersions(a: string, b: string) {
```

Comment:
The expected usage of this function is a little confusing to me. (I.e., is it for sorting? Exact version matches? Etc.) The fact that `localeCompare` does _roughly_ the right thing for some version strings seems a little shaky, esp. given that this runs in the local environment's default locale.

I also don't think it does exactly what we want in even some pretty natural cases:

```
// good!
> "1.15.0".localeCompare("1.14.0")
1
// less good
> "1.15.0".localeCompare("1.2.9")
-1
```

Might be less fiddly to just use the [semver](https://www.npmjs.com/package/semver) library to do explicit version matching instead.

Reply:
i want to delete this entirely

Reply:
i went back and forth on the library. i'll add the library

Reply:
OK that's added. 

by the way 
```
"1.15.0".localeCompare("1.14.0", undefined, { numeric: true, sensitivity: 'base' })
1
"1.15.0".localeCompare("1.2.9", undefined, { numeric: true, sensitivity: 'base' })
1 
```